datastream-server/src/test/java/com/linkedin/datastream/server/assignment/TestBroadcastStrategy.java

```diff
@@ -279,4 +280,21 @@ public void testRebalanceTasksWhenNewInstanceIsAdded() {
 
     Assert.assertEquals(newAssignment.get(instance4).size(), expectedNumTasksPerInstance);
   }
+
+  @Test
+  public void testDontCreateNewTasksWhenInstanceChanged() {
```

Comment:
Rename to something along the lines of: `testReuseDeadInstanceTasks`?

Reply:
kept the test name same as StickyMulticastStrategy.

Reply:
No problem